src/docusaurus.config.js

```diff
@@ -61,6 +62,7 @@ const config = {
         theme: {
           options: {
             scrollYOffset: 60,
+            disableSearch: true,
```

Comment:
hahahah nice i knew it

Reply:
yeah! It was really painful to find the right spot for this haha